pkg/ocm/helpers.go

```diff
@@ -1122,3 +1122,17 @@ func ValidateClaimValidationRules(input interface{}) error {
 	}
 	return nil
 }
+
+// AutoNodeExists checks if AutoNode is enabled on the cluster
+func AutoNodeExists(cluster *cmv1.Cluster) bool {
+	return cluster.AutoNode() != nil && cluster.AutoNode().Mode() == "enabled"
+}
```

Comment:
I left a comment on the original changes, but it seems this still hasn't changed. I would prefer if this were different

1. `enabled` is only one possible value for autonode
2. This only returns a bool, we will only get `false` in a case where `cluster.AutoNode()` is `nil`

As stated on the original comment, I think we should use a more Go-style `exists` function pattern.

Take a look at `isHcpBillingTechPreview` in the OCM Client. Notice how we are accomplishing the same thing, but we get a lot more useful information out of it with that error return

To add onto the first point: in the future, what would happen if the auto node value is valid, not `enabled`? We would always get `false` here